databases.go

```diff
@@ -0,0 +1,607 @@
+package godo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	databaseBasePath        = "/v2/databases"
+	databaseSinglePath      = databaseBasePath + "/%s"
+	databaseResizePath      = databaseBasePath + "/%s/resize"
+	databaseMigratePath     = databaseBasePath + "/%s/migrate"
+	databaseMaintenancePath = databaseBasePath + "/%s/maintenance"
+	databaseBackupsPath     = databaseBasePath + "/%s/backups"
+	databaseUsersPath       = databaseBasePath + "/%s/users"
+	databaseUserPath        = databaseBasePath + "/%s/users/%s"
+	databaseDBPath          = databaseBasePath + "/%s/dbs/%s"
+	databaseDBsPath         = databaseBasePath + "/%s/dbs"
+	databasePoolPath        = databaseBasePath + "/%s/pools/%s"
+	databasePoolsPath       = databaseBasePath + "/%s/pools"
+	databaseReplicaPath     = databaseBasePath + "/%s/replicas/%s"
+	databaseReplicasPath    = databaseBasePath + "/%s/replicas"
+)
+
+// DatabasesService is an interface for interfacing with the databases endpoints
+// of the DigitalOcean API.
+// See: https://developers.digitalocean.com/documentation/v2#databases
+type DatabasesService interface {
+	List(context.Context, *ListOptions) ([]Database, *Response, error)
+	Get(context.Context, string) (*Database, *Response, error)
```

Comment:
Ugh. Can you give me the reasoning for why we're using `Get` and `GetDB` vs `Describe` and `Get`?

Reply:
Ok, make sense to me now